### DISCUSSION TITLE
Nowe konto firmowe... od czego zacząć.

### DISCUSSION THREAD

**Discussion by pluciorx** [AzureCommunityPL/group#424] [Ogólna]
Korzystam z azure od paru lat prywatnie, teraz chciałbym założyć sobie zupełnie nową subskrypcję na firmę, do tego firmowy DevOps is office 365 z firmową domena, czy tylko dla mnie ten proces jest nie jasny? ma ktoś może jakies opisy jak to ugryźć ? 
z tego co zrozumiałem to z konta prywatnego w Azure AD tworzy się nowego Tenanta, a jak wtedy z Office 365 ? chciałbym od początku uniknąć chaosu z subskrypcjami. 
Help...

**Comment by JanuszNowak** [comment]
Hej@pluciorx 

Organizacje Azure DevOps tworzysz przez [portal](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops)
Stworzono organizacje [podpinasz](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/connect-organization-to-azure-ad?view=azure-devops) pod tenant Azure Active Directory 
Do organizacji ADO podpinasz [billing](https://docs.microsoft.com/en-us/azure/devops/organizations/billing/set-up-billing-for-your-organization-vs?view=azure-devops), wybierasz subskrypcje Azure, pod którą ewentualnie będą pobierane możliwe koszta dodatkowych licencji, build agentów itd.

Azure active directory (tetant) [związujesz](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory#associate-a-subscription-to-a-directory)  z subskrypcją Azure.

[Powiązanie](https://github.com/uglide/azure-content/blob/master/articles/billing-add-office-365-tenant-to-azure-subscription.md) rachunku Office 365 do subskrypcji Azure.
[Powiązanie](https://redmondmag.com/articles/2017/05/01/connect-azure-ad-to-office-365.aspx) Azure Active directory do O365 
lub [powiązanie ](https://docs.microsoft.com/en-us/microsoft-365/education/deploy/intro-azure-active-directory)

Mała uwaga:
Podczas tworzenia dzierżawy usługi Azure AD należy określić region logiczny, który określi lokalizację centrum danych. Należy to wybrać bardzo ostrożnie, ponieważ nie można tego zmienić po stworzeniu.

**Reply by wi5nia** [reply]
Możesz po polsku napisać @JanuszNowak ?

**Reply by JanuszNowak** [reply]
@wi5nia done ;) pisałem wcześniej z telefonu jadąc na hulajnodze ;)

**Reply by wi5nia** [reply]
Czyli tworzyłeś zagrożenie w ruchu lądowym...

**Reply by JanuszNowak** [reply]
@wi5nia i tak i nie, jechałam w prywatnym ogrodzonym lesie/polu ;) hulajnoga była od Tesli (seld driving), i świadomie nie podaje lokalizacji, gdzie miało to miejsce (regulacje danych krajów).

**Reply by wi5nia** [reply]
Co o takiej godzinie robiles na hulajnodze na prywatnym terenie? Nie powinienes w tym czasie oszczedzac milionow dla firmy?

**Reply by JanuszNowak** [reply]
@wi5nia miliony już zaoszczędzone i budowanie wartości i eliminacja "waste" trwa cały czas "continusly". Dużo obszarów do poprawy :) Nie wiem jak u ciebie, u mnie praca zdalna cały czas jest możliwa. Czas dla #community zawsze się znajdzie, jak nie tutaj to jest dużo innych. @wi5nia trzymaj się ciepło, musimy się spotkać na hulajnogach 👍

**Reply by smereczynski** [reply]
> [Powiązanie](https://github.com/uglide/azure-content/blob/master/articles/billing-add-office-365-tenant-to-azure-subscription.md) rachunku Office 365 do subskrypcji Azure. [Powiązanie](https://redmondmag.com/articles/2017/05/01/connect-azure-ad-to-office-365.aspx) Azure Active directory do O365 lub [powiązanie ](https://docs.microsoft.com/en-us/microsoft-365/education/deploy/intro-azure-active-directory)

Te poradniki to prehistoria. Przepis jest prosty. Tworzysz nowy tenant AAD i budujesz nową organizację w ujęciu chmury MS. Koniec przepisu.

**Comment by smereczynski** [comment]
Żeby utworzyć tenant AAD idź pod ten adres: https://account.azure.com/organization (nie ma go w dokumentacji, a oszczędzi Ci walkę z prywatnym kontem MS)
